### PR TITLE
Replace Buba/Babel with TypeScript

### PR DESCRIPTION
**Pull request by hoichi**
Another fix for https://github.com/mostjs-community/last/issues/1

Here it is, I guess.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  end (t: Time) {
+    if (this.held) {
+      this.sink.event(t, this.held.val)
+    }
+    this.sink.end(t)
+  }
+}
+
+export { last, Last }
```
Comment (hoichi) [inline src/index.ts]: The only obvious issue I have is that the `--declaration` flag in tsconfig forces me to export `Last` along with `last`. Not sure if it’s OK.

Comment (TylorS) [reply src/index.ts]: Hey @hoichi, this looks great! Regarding the export of Last, you can remove it by adding a `: Stream<T>` return type to the `last` function.

**Comment by hoichi** [comment]
@TylorS fixed the class export, thanks for the tip!

Also bumped minor version. In theory, migration to TS shouldn’t have changed anything, but who knows :grin:

**Comment by TylorS** [comment]
This looks great to me 👍

**Comment by TylorS** [comment]
Thank you for your work here @hoichi

**Comment by hoichi** [comment]
You’re welcome. Thank you for all the tips!

**Pull request merged**